packages/google-closure-compiler/cli.js

```diff
@@ -83,116 +83,27 @@ Object.keys(compilerFlags).forEach(flag => {
   }
 });
 
-if (platform !== 'javascript') {
-  const Compiler = require('./lib/node/closure-compiler');
-  let args = process.argv.slice(2);
-  for (let i = 0; i < args.length; i++) {
-    if (/^--platform/.test(args[i])) {
-      let delCount = 1;
-      if (args[i].indexOf('=') < 0 && args.length > i + 1) {
-        delCount++;
-      }
-      args.splice(i, delCount);
-      break;
+const Compiler = require('./lib/node/closure-compiler');
+let args = process.argv.slice(2);
+for (let i = 0; i < args.length; i++) {
+  if (/^--platform/.test(args[i])) {
+    let delCount = 1;
+    if (args[i].indexOf('=') < 0 && args.length > i + 1) {
```

Comment:
Nit: is `includes` avialable?

Reply:
Yes - but that would be a change outside this PR. I would like to modernize the code base at some point including utilizing async/await which is also available.